cmd_clean/main.go

```diff
@@ -53,7 +53,9 @@ func main() {
 	cf.SsoDomain = vcSsoDomain
 	cf.HttpsPort = vcHttpsPort
 	cf.Thumbprint = []string{thumbprint}
```

Comment:
thumbprint could also be nil, thumbprint and nsx ca cert are two options

Reply:
Oh, there is 
	nsxConfig.Thumbprint = removeEmptyItem(nsxConfig.Thumbprint)
	nsxConfig.CaFile = removeEmptyItem(nsxConfig.CaFile)
in validateCert, can remove the `if` here.